sources/packages/backend/apps/db-migrations/src/sql/SFASIndividuals/Add-ppd-status-ppd-status-date.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE
+    sims.sfas_individuals
+ADD
+    COLUMN ppd_status BOOLEAN;
```

Comment:
@andrewsignori-aot @guru-aot @andrepestana-aot  I have a confusion here, should we add a new column or  should we change the existing coulmn type to enum on sfas_individuals.?

Reply:
As far as I understood `sims.sfas_individuals` just keeps the raw data that comes from the integration.

Reply:
I agree with @andrepestana-aot. The sfas* tables can reflect as much as possible the raw data received.

Reply:
Yes i believe that we can have the raw data what is received.